bulbs/utils/methods.py

```
@@ -99,3 +101,9 @@ def get_overridable_template_name(parent_name, child_name):
         template_name = parent_name
 
     return template_name
+
+def get_video_object(video_id):
+    video_model_name = getattr(settings, "VIDEO_MODEL", "")
```

Should fail/error if `VIDEO_MODEL` is not defined